sdk/src/assertion.rs

```diff
@@ -471,12 +471,12 @@ impl Assertion {
     }
 }
 
```

Comment:
Maybe add just an unused tag so the code is still available.  I think this is used by another PR somewhere.